### ISSUE TITLE
the problem of batchsize and output[-1]?

### ISSUE THREAD

**Issue by ky-xt** [polyaxon/traceml#9]
sorry, may be I need your help again. I have some problems as follow:
1.your code：
**x_ = tf.unpack(X, axis=1, num=num_units)
output, layers = tf.nn.rnn(stacked_lstm, x_, dtype=dtypes.float32)**
the shape of the x_ is (**size**,1)  but the **size** is not equal to **batch_size**.
However, the API doc show the function receives the parameters with the shape of  **[batch_size, input_size]**
tf.nn.rnn(cell, inputs, initial_state=None, dtype=None, sequence_length=None, scope=None) 
**Args:**
    _cell_: An instance of RNNCell.
    _inputs_: A length T list of inputs, each a Tensor of shape **[batch_size, input_size]**, or a nested tuple of such elements.
why is it?
2. Another problem
code:
**output = dnn_layers(output[-1], dense_layers)**
why is output[-1] rather than output? Awaiting for your @mouradmourafiq reply. Thanks

**Comment by mouradmourafiq** [comment]
I am not sure I understand how you concluded that `x_ is (size,1)`. But for the question number 2 we just need the last prediction from out inputs, you can have more insights on how lstm works from this [post](http://colah.github.io/posts/2015-08-Understanding-LSTMs/).

**Comment by ky-xt** [comment]
谢谢，辛苦了，已收到你的邮件。
